src/main/java/com/internship/frejaeidjmeterplugin/jmeter/sampler/GeneralSampler.java

```diff
@@ -0,0 +1,93 @@
+package com.internship.frejaeidjmeterplugin.jmeter.sampler;
+
+import com.internship.frejaeidjmeterplugin.jmeter.frejaRequests.AuthenticationService;
+import com.internship.frejaeidjmeterplugin.jmeter.frejaRequests.SignService;
+import com.verisec.frejaeid.client.beans.authentication.get.AuthenticationResult;
+import com.verisec.frejaeid.client.beans.sign.get.SignResult;
+import com.verisec.frejaeid.client.enums.MinRegistrationLevel;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientInternalException;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientPollingException;
+import com.verisec.frejaeid.client.exceptions.FrejaEidException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.apache.jmeter.samplers.AbstractSampler;
+import org.apache.jmeter.samplers.Entry;
+import org.apache.jmeter.samplers.SampleResult;
+
+public class GeneralSampler extends AbstractSampler {
```

Comment:
I would expect from this class to use AuthSampler and SignSampler regarding which checkbox is active, not initiating services again. 
This way we have duplicated logic because we have already called authService and signService methods there.